index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>Portfolio-Project</title>
   </head>
   <body>
```

Comment:
- [ ] Heading levels should have a meaningful hierarchy, please consider using the ```<h1>``` tag only one time preferably for the page header, and use ```<h2>``` to ```<h6>``` in descending way, so you can respect the hierarchy of the ```hx``` tags